api/src/config.js

```diff
@@ -2,13 +2,70 @@ const _ = require('underscore'),
       db = require('./db-pouch'),
       ddocExtraction = require('./ddoc-extraction'),
       translations = require('./translations'),
-      defaults = require('./config.default.json'),
+      defaults = require('../../config/standard/app_settings.json'),
```

Comment:
I still think this won't work in production as the `config` directory won't exist. We'll need to bundle the default config up and attach it to the ddoc just like the forms.